### ISSUE TITLE
json: markers reported on start for files not opened

### ISSUE THREAD

**Issue by vince-fugnitto** [eclipse-theia/theia#7154]
**Description**

When `theia` is used as a workspace, I noticed that `json` warnings are outputted (`.theia`), despite any of the files being opened. Is this the intended behavior? When verifying the same use case in VS Code, no warnings are present in the `problems` view:

![image](https://user-images.githubusercontent.com/40359487/74536284-c793bb80-4f05-11ea-9bb8-637caa910a31.png)

**Steps to Reproduce**
1. start the application (better to start with a fresh port and no local storage)
2. open `theia` as a workspace (no files should be opened)
3. the `problems view` outputs warnings for **json** files under `.theia`

**Comment by akosyakov** [comment]
@vince-fugnitto probably we store markers between sessions, do you think it is bad? misleading? or helpful since it shows you what is really wrong faster?

**Comment by vince-fugnitto** [comment]
> @vince-fugnitto probably we store markers between sessions, do you think it is bad? misleading? or helpful since it shows you what is really wrong faster?

I felt that, especially since they are warnings in this case, that displaying the markers on startup is a bit noisy. I noticed in VS Code this same behavior is not present, the markers are only visible when first opening the file and not on startup. I am surprised it'd store the markers between sessions, I made sure to start at a different port and also delete my local history. That is of course my own opinion, I wonder what others might think.

cc @marcdumais-work @lmcbout @marechal-p
